net/tsdial/tsdial.go

```diff
@@ -337,9 +338,10 @@ func (d *Dialer) UserDial(ctx context.Context, network, addr string) (net.Conn,
 		}
 		return d.NetstackDialTCP(ctx, ipp)
 	}
-	// TODO(bradfitz): netns, etc
-	var stdDialer net.Dialer
-	return stdDialer.DialContext(ctx, network, ipp.String())
+	if tsaddr.IsTailscaleIP(ipp.Addr()) {
```

Comment:
Better than what it was but it doesn't work for subnet routes or when using exit nodes.

Reply:
Leave a todo?

Reply:
Done, and changed commit message.